source/scss/style.scss

```diff
@@ -87,5 +87,8 @@
 // Form Checkbox
 @import "components/form-checkbox";
 
+// Brand Colors
+@import "components/colors";
```

Comment:
Since we don't need the component, but the variables, this can be removed as well.